### PR TITLE
Add rake task for publishing special routes.

### PR DESCRIPTION
**Pull request by elliotcm**
Registers `/government` and `/courts-tribunals`.

https://trello.com/c/blLdEZN5/292-make-apps-register-special-routes-on-deploy

### CODE REVIEW

File: test/unit/special_route_test.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+require "gds_api/publishing_api"
+
+class SocialMediaServiceTest < ActiveSupport::TestCase
```
Comment (edds) [inline test/unit/special_route_test.rb]: Should this be `SpecialRouteTest`?

Comment (elliotcm) [inline test/unit/special_route_test.rb]: Yes, updated.

**Comment by alext** [comment]
This will also require changes to the deploy hooks to run the new task instead of the old router task.

**Comment by elliotcm** [comment]
@alext Yes, they all will.

**Comment by danielroseman** [comment]
:+1:

**Comment by elliotcm** [comment]
@danielroseman Updated to use the gem, and add the rake task alias.  Could you recheck?

**Comment by danielroseman** [comment]
LGTM

**Pull request merged**